test/es6/rlexe.xml

```diff
@@ -121,13 +121,15 @@
       <tags>BugFix</tags>
     </default>
   </test>
+  <!-- OS BUG 7117880
```

Comment:
Since the compiler issue will affect any builds in MicroBuild v2 this could be a problem for release/1.1 as well. Not sure if it's strictly necessary.